### PR TITLE
Change names and add checks for command line arguments in ThemeBuilder

### PR DESCRIPTION
**Pull request by babich-a**

### CODE REVIEW

File: themebuilder/package.json [hunk 1]
```diff
@@ -19,8 +19,7 @@
     "sass": "^1.10.3"
   },
   "bin": {
-    "build-theme": "cli/build-theme.js",
-    "build-base-theming": "cli/build-base-theming.js"
+    "devextreme-themebuilder": "cli/index.js"
```
Comment (BingoRUS) [inline themebuilder/package.json]: We don't need commands to be available for this npm package. Let's remove it.

**Review by BingoRUS — CHANGES_REQUESTED** [review]

**Review by BingoRUS — DISMISSED** [review]
👍

**Comment by babich-a** [comment]
Sure. I also add info about swatch selector will be used. Check it, please.

**Review by BingoRUS — APPROVED** [review]
👌

**Pull request merged**